### ISSUE TITLE
Unavailability of QDTK

### ISSUE THREAD

**Issue by chris-ap** [f-koehler/mlxtk#156]
I'm very interested in ML-MCTDH-X simulations and I tried to install this toolkit. The problem is: it depends on the QDTK package, but I couldn't find QDTK anywhere on the web. Where can I find and install QDTK?

**Comment by f-koehler** [comment]
Thank you very much for your interest in this project. So a bit of background:
QDTK is part of the ML-MCTDHX Package that we develop in the group of Prof. Schmelcher in Hamburg. It is not a complex library and is used only to create operator and wave function files. The actual propagation is then performed using a complex Fortran software package.

Unfortunately, both QDTK as well as the Fortran code are closed source at the moment and will probably remain that way in the foreseeable future. The reason for this is twofold:

- We are afraid of misuse of the method that would shed bad light on it. We have seen this with other methods.
- The documentation is not good enough to hand it out. There are many things one can do wrong when using this method. This could lead to inaccurate results and could contribute two my first point.

mlxtk is a library which makes the work with the ML-MCTDHX simpler, more convenient and efficient. It is my personal project and I decided it to develop it public and have easy access to it from everywhere as well as to share it with the rest of the group. I might want to add a disclaimer in the README.

Even though I am currently also the main and only developer of our ML-MCTDHX implementation I cannot just decide to open source it. Sorry!

If you are really interested however, there are ways to gain access. We hand out the software to scientific collaborators (I do not know your status) and students (bachelor, master and PhD) in our group but always in close collaboration such that we can ensure the proper use of the method.

**Comment by chris-ap** [comment]
Thank you very much for your exhaustive explanation!
I am a PhD student from Università degli Studi di Milano (Milan, Italy), and I'll talk with my tutor about this. In the meanwhile, I'd like to ask you a question. The kind of application of ML-MCTDHX we are interested in is the study of the motion of a few (one or two) impurities in a fermionic bath, made of a relatively large number (at least a dozen; but a few dozens would be much better)  of weakly-interacting particles. Our hope is being able to describe the effect of the impurity (or impurities) on the bath, and, if it is feasible, also the effective interaction between two impurities.
Do you think that such a study would be feasible with your software?

**Comment by f-koehler** [comment]
Yes, that is definitely something you can do. See for example [this work](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.101.053619) which investigates the correlated dynamics of fermionic impurities immersed in a Fermi sea. Most of our works focus on bosonic particles however (e.g. https://iopscience.iop.org/article/10.1088/1367-2630/ab9e34). You can find many more works [here](https://www1.physik.uni-hamburg.de/en/ilp/schmelcher/publications.html).

I also want to point you to one last [reference](https://aip.scitation.org/doi/10.1063/1.4993512) that should give you a good idea how the algorithm works and what you can investigate with it.
